mlir/lib/Dialect/SCF/Transforms/TileUsingInterface.cpp

```diff
@@ -1979,6 +2033,26 @@ mlir::scf::tileAndFuseConsumerOfSlice(RewriterBase &rewriter,
         consumerOp, "consumer op's operand doesn't seem to be an OpResult");
   }
 
+  SmallVector<OpOperand *> potentialOperands = {*maybeConsumerOpOperand};
```

Comment:
Please leave some comments as to what this is for.

Reply:
The comment below actually is fine. Move that up above these.
Also instead of a `SmallVector<OpOperand *>` and a `SmallVector<Operation *>` for the slices, just make a `SmallVector<std::tuple<OpOperand *, Operation *>>`